### ISSUE TITLE
Training about LIP

### ISSUE THREAD

**Issue by SwagJzzZ** [PkuRainBow/OCNet.pytorch#34]
Hi @PkuRainBow ,

I noticed that you have trained your network on LIP dataset. Can you provide the details about the data augmentation and the loss setting.

Best wishes!

**Comment by PkuRainBow** [comment]
We will release the code on LIP in the future. Please keep waiting.

**Comment by zzw1123** [comment]
Hi, @PkuRainBow , thanks for the open source work!
When training the baseline, do you use the same parameters(e.g. learning rate, optimizer) as training asp_oc?
